lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -1647,35 +1671,130 @@ namespace Js
 
         return Js::JavascriptString::NewCopySz(strBuf, scriptContext);
 #else
-        // TODO (doilij): implement INTL_ICU version
-#ifdef INTL_ICU_DEBUG
-        Output::Print(_u("EntryIntl_FormatDateTime > returning null, fallback to JS\n"));
+        INTL_CHECK_ARGS(
+            args.Info.Count == 4 &&
+            DynamicObject::Is(args.Values[1]) &&
+            (TaggedInt::Is(args.Values[2]) || JavascriptNumber::Is(args.Values[2])) &&
+            JavascriptBoolean::Is(args.Values[3])
+        );
+
+        DynamicObject *state = DynamicObject::UnsafeFromVar(args.Values[1]);
```

Comment:
>UnsafeFromVar [](start = 46, length = 13)

add comment on the first one to wit "unsafe because we already checked the type above"